packages/nimble-components/src/table/models/table-validator.ts

```diff
@@ -130,6 +143,11 @@ export class TableValidator<TData extends TableRecord> {
         return !this.duplicateGroupIndex;
     }
 
+    public validateColumnConfigurations(columns: Validatable[]): boolean {
+        this.invalidColumnConfiguration = columns.some(x => !x.checkValidity());
+        return !this.invalidColumnConfiguration;
+    }
```

Comment:
This data flow seems backwards to me. The intention was to behave like the other columnInternals properties. They report their changes by being observable, the table does not ask them to be calculated.

Notice for example how column elements listen for their own property changes and then set their columnInternals properties. I'd expect the same here, a column would be responsible for it's own property changes and set the columnInternals.validationConfiguration property just by monitoring it's own property changes. It would not rely on the table calling checkValidity.

Reply:
Okay, but how would the table validator's `invalidColumnConfiguration` flag get set, if not by the table validator querying each column's validity? This seems to be what other `validate*` functions in the table validator do. (Note I do not expect `checkValidity()` to actually calculate anything, but just return the last calculated value.)

Reply:
Done